code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -381,8 +387,154 @@
 
 	suicide_messages = list(
 		"is melting into a puddle!",
-		"is turning a dull, brown color and melting into a puddle!",
-		"is ripping out their own core!")
+		"is ripping out their own core!",
+		"is turning a dull, brown color and melting into a puddle!")
+
+	var/list/mob/living/carbon/human/recolor_list = list()
+
+	species_abilities = list(
+		/mob/living/carbon/human/verb/toggle_recolor_verb,
+		/mob/living/carbon/human/proc/regrow_limbs
+		)
+
+/datum/species/slime/handle_life(var/mob/living/carbon/human/H)
+	var/const/color_shift_trigger = 0.1
+	var/const/icon_update_period = 600 // 1 minute
+	var/const/blood_scaling_factor = 5 // Used to adjust how much of an effect the blood has on the rate of color change. Higher is slower.
```

Comment:
This way, they're right next to the place they're used - I GUESS I could use a #define, but then I'd need to worry about #undefining them too on the other side of the function, and so would future coders, if I wanted to avoid polluting the preprocessor's namespace